cloud/openssh/sshshim/generate.go

```diff
@@ -0,0 +1,67 @@
+package sshshim
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"go.jetpack.io/devbox/boxcli/featureflag"
+	"go.jetpack.io/devbox/cloud/openssh"
+)
+
+const (
+	configShimDir = ".config/devbox/ssh/shims"
+)
+
+// Setup creates the ssh and scp symlinks
+func Setup() error {
+	if featureflag.SSHShim.Disabled() {
+		return nil
+	}
+	shimDir, err := Dir()
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	if err := openssh.EnsureDirExists(shimDir, 0744, true /*chmod*/); err != nil {
+		return err
+	}
+
+	// TODO use in the next PR instead of sshExecutablePath
+	//devboxExecutablePath, err := os.Executable()
+	//if err != nil {
+	//	return errors.WithStack(err)
+	//}
+	sshExecutablePath, err := exec.LookPath("ssh")
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	if err := os.Symlink(sshExecutablePath, filepath.Join(shimDir, "ssh")); err != nil {
+		if !os.IsExist(err) {
```

Comment:
Should the symlink always be overwritten just in case the devbox path changed?

Reply:
Yeah, I also ran into it because the ssh symlink needed to be updated from /usr/bin/ssh to point to the devbox binary (when testing Part 1 and then Part 2 PRs) 

I'll make this fix.

Reply:
fixed in part 2 PR: https://github.com/jetpack-io/devbox/pull/349/commits/077eeb274bac167f7ab6ac12050f928ee220314f